ra/ra_test.go

```diff
@@ -3348,6 +3348,7 @@ func TestCTPolicyMeasurements(t *testing.T) {
 }
 
 func TestWildcardOverlap(t *testing.T) {
+	_ = features.Set(map[string]bool{"EnforceOverlappingWildcards": true})
```

Comment:
It might be worth adding a `defer features.Reset()` here. I think setting this will affect other tests in the RA otherwise.